MFRC522.h

```diff
@@ -78,6 +78,8 @@
 #include <Arduino.h>
 #include <SPI.h>
 
+#define MFRC522_SPICLOCK (F_CPU / 2)			// Fastest possible speed (MFRC522 accept upto 10MHz)
```

Comment:
(1)  Preprocessor are old and ugly. I would prefer an internal var, with public setter.
(2) `(F_CPU / 2)` maybe break compatibility with alternative boards. Default should be still `SPI_CLOCK_DIV4`. (can be changed with (1))